public/app/modules/home/home.html

```diff
@@ -0,0 +1,138 @@
+<div class="container-fluid">
+	<!-- NAV BAR -->
+	<div class="row" ng-include="'app/modules/partials/navbar.html'"></div>
+	<!-- CAROUSEL -->
+	<div class="row" ng-include="'app/modules/partials/carousel.html'">
+
+	</div>
+</div>
+<div class="container">
+	<div class="row">
+		<div class="col-sm-12 col-md-4 col-lg-4">
+			<div class="row">
+				<div class="col-sm-6 col-md-12 col-lg-12">
+					<div class="box box-4col box-1" style="background-image: url(/images/post_its.jpg);">
+						<div class="box-caption">
+							<div class="label label-text-2 black-text">IDEAS</div>
+                                    <span class="title-2 white-text">
+                                        A Caribbean Open Data Public Service
```

Comment:
@tremainebuchanan I am guessing all of the fixed/hardcoded stuff will be changed?